apikit-tools/apikit-scaffolder/src/main/java/org/mule/tools/apikit/output/scopes/ConsoleFlowScope.java

```diff
@@ -31,11 +28,7 @@ public ConsoleFlowScope(Element mule, API api, String configRef, String httpList
 
         if (httpListenerConfigRef != null)
```

Comment:
there is no need to have this "if" anymore. The method will always receive a listener config.